application/controllers/collection_instrument.py

```diff
@@ -125,19 +125,22 @@ def upload_to_bucket(self, exercise_id, file, ru_ref=None, classifiers=None, ses
     @staticmethod
     def validate_non_duplicate_instrument(file, exercise_id, session):
         exercise = query_exercise_by_id(exercise_id, session)
-
+        log.info("Validating if instrument is already uploaded for this exercise", exercise_id=exercise_id)
```

Comment:
Just a bit too much logging, there is so much across our repo I don't think we need function entry